### PR TITLE
Txn status

### PR DESCRIPTION
**Pull request by Prannav-k**

### CODE REVIEW

File: app/core/blockchain/state_updater.py [hunk 1]
```diff
@@ -47,13 +46,17 @@ def update_db_states(cur, block):
...
                 handle_txn(cur, transaction,newblockindex,newblockindex)
+            transaction['status'] = 1     
         except Exception as e:
             txn_code = transaction["transaction"]['trans_code']
             logger.error(f'Error in transaction: {str(txn_code)}')
             logger.error(str(e))
-            logger.error(traceback.format_exc()) 
+            logger.error(traceback.format_exc())
+            transaction['status'] = 2     
+            transaction['exec_msg'] = str(e)
```
Comment (kousthubraja) [inline app/core/blockchain/state_updater.py:56]: Limit exec_msg to first 100 characters.

File: tests/test_sample_template.py [hunk 2]
```diff
@@ -174,3 +164,50 @@ def test_invlaid_sc_call(request):
...
+    response = requests.post(
+        NODE_URL+'/validate-transaction', json=signed_transaction)
+    
+    assert response.json()["response"]["valid"] == True
+
+    if TEST_ENV == 'local':
+        response=requests.post(
+            NODE_URL + '/run-updater?add_to_chain_before_consensus=true')
+    else:
+        print('Waiting to mine block')
```
Comment (kousthubraja) [inline tests/test_sample_template.py:211]: check with `get-transaction` API.

File: app/core/blockchain/state_updater.py (hunk 1 above)
Comment (Prannav-k) [reply app/core/blockchain/state_updater.py]: added in latest commit

File: tests/test_sample_template.py (hunk 2 above)
Comment (Prannav-k) [reply tests/test_sample_template.py:211]: it is working

Comment (Prannav-k) [reply tests/test_sample_template.py:211]: new assertion is added using get-txn api

**Pull request merged**